features/facebookImport/src/model/entities/facebook-account.js

```diff
@@ -1,8 +1,11 @@
+import Entity from "./entity.js";
 import MessageThreadsGroup from "./message-threads-group.js";
 import RelatedAccountsGroup from "./related-accounts-group.js";
 
-class FacebookAccount {
+class FacebookAccount extends Entity {
```

Comment:
Again, this is asking for a more general level, er, entity, and maybe we should step back and discuss where we want to put it. FacebookAccount should be the lowest level of abstraction we should handle at the feature level. If we want anything else, which is more low-level, we should probably try and ponder different alternatives for it.